eng/testing/tests.mobile.targets

```diff
@@ -162,6 +162,23 @@
       AssemblySearchPaths="@(AssemblySearchPaths)" />
   </Target>
 
+  <UsingTask TaskName="TimeZoneLoader"
+             AssemblyFile="$(WasmBundleTaskTasksAssemblyPath)" />
+  <Target Condition="'$(TargetOS)' == 'Browser'" Name="LoadTimeZone" >
```

Comment:
Place the name before the condition.
And maybe rename the target to `LoadTimeZoneData`?

Reply:
Just following the ordering of other targets.